common/remote_util.h

```diff
@@ -38,44 +39,48 @@ class RemoteUtil {
   RemoteUtil(int verbosity, bool quiet, ProcessFactory* process_factory,
              bool forward_output_to_log);
 
-  // Returns the initialization status. Should be OK unless in case of some rare
-  // internal error. Should be checked before accessing any members.
-  const absl::Status& GetInitStatus() const {
-    return sdk_util_.GetInitStatus();
-  }
+  // Sets the hostname of the remote instance and the ssh tunnel port. The
+  // hostname can be an IP address and/or contain a username, e.g. user@1.2.3.4.
+  void SetHostAndPort(std::string hostname, int ssh_port);
```

Comment:
Can you change this to any of `Set(Remote|Target|Dest)HostAndPort` to make the direction clearer?

Reply:
Renamed to SetUserHostAndPort() for consistency.